tests/testthat/test-aes.R

```diff
@@ -48,8 +48,10 @@ test_that("aes evaluated in environment where plot created", {
   df <- data_frame(x = 1, y = 1)
   p <- ggplot(df, aes(foo, y)) + geom_point()
 
-  # Accessing an undefined variable should result in error
-  expect_error(get_layer_data(p), "'foo' not found")
+  test_that("accessing an undefined variable results in an error", {
+    skip_if(getRversion() <= "4.4.0")
+    expect_snapshot(get_layer_data(p), error = TRUE)
+  })
```

Comment:
Is it intended you put a `test_that()` call inside another `test_that()` call?

Reply:
Yes, that way I can conditionally skip a single expectation without throwing out all other expectations in a test. I improvised this to skip some snapshots on older R versions. This was due to error formatting that has changed across R versions.